sources/api/controllers/GroupController.ts

```diff
@@ -227,3 +227,13 @@ export async function kickUserFromGroup(req: Request, res: Response) {
         ]
     });
 }
+
+export async function getGroupCalendar(req: Request, res: Response) {
```

Comment:
I was thinking of adding a `calendar` field in the `GET /groups/{group-id}` response to make this information directly accessible

Reply:
this route might be useful in the future, when calendars will become more complicated, we need to discuss this route later

Reply:
Will do